distribution-spec/v1.0/django-oci/report.html

```diff
@@ -193,15 +193,15 @@ <h1>OCI Distribution Conformance Tests</h1>
       </tr>
       <tr>
         <td class="bullet-left">Start Time</td>
-        <td>Oct 11 23:01:22.334 -0600 MDT</td>
+        <td>Dec 4 21:08:51.475 &#43;0000 UTC</td>
```

Comment:
and another encoding issue.. I wonder if related to a dependency in that repo. Its maybe security related?

Reply:
I think it's just for HTML encoding https://www.w3.org/MarkUp/html-spec/html-spec_13.html akin to how the browser URL will do that too, and yes could be for security.